MemoryLeak.Dockerfile

```
@@ -1,23 +1,24 @@
 FROM node:23.11.1-alpine3.21
 
 RUN apk add --no-cache \
```

Hadolint (2.12.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)